packages/utils/src/route.js

```diff
@@ -41,13 +41,13 @@ export const flatRoutes = function flatRoutes (router, fileName = '', routes = [
   return routes
 }
 
-function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash, namePrefix) {
+function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash, parentRouteName) {
   const regExpIndex = new RegExp(`${routeNameSplitter}index$`)
```

Comment:
Not a new code but I belive this could be like some and there would be no need for `route.name === 'index'` check below.
```suggestion
  const regExpIndex = new RegExp(`(^|${routeNameSplitter})index$`)
```

Reply:
`regExpIndex` is used to remove the last occurrence of `-index from `route.name` in line 77, this change will break the replacement

Reply:
OK, never mind. :)